src/app/space/create/deployments/deployments-toolbar/deployments-toolbar.component.less

```diff
@@ -0,0 +1,5 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+
+.filter-fields .form-group .form-control {
+  width: 14em;
```

Comment:
If we increase the system font size, this will increase as well. Is that intended?

Reply:
Sure, I guess? The text input will scale up along with the text's size - I don't know if that's right or wrong, but it sounded reasonable in my head.

Reply:
Alrighty, sounds good. Just wanted to make sure. Approving the PR.